client/my-sites/plugins/jetpack-plugins-setup/index.jsx

```diff
@@ -263,73 +267,94 @@ const PlansSetup = React.createClass( {
 	},
 
 	renderStatus( plugin ) {
+		if ( plugin.error ) {
+			return this.renderStatusError( plugin );
+		}
+
 		const { translate } = this.props;
 		const statusProps = {
 			isCompact: true,
 			status: 'is-info',
 			showDismiss: false,
+			icon: 'plugins',
 		};
 
-		if ( plugin.error ) {
-			statusProps.status = 'is-error';
-			switch ( plugin.status ) {
-				case 'install':
-					statusProps.text = translate(
+		switch ( plugin.status ) {
+			case 'done':
+				// Done doesn't use a notice
+				return (
+					<div className="plugin-item__finished">
+						{ translate( 'Successfully installed & configured.' ) }
+					</div>
+				);
+			case 'activate':
+			case 'configure':
+				return <Notice { ...statusProps } text={ translate( 'Almost done' ) } />;
+			case 'install':
+				return <Notice { ...statusProps } text={ translate( 'Working…' ) } />;
+			case 'wait':
+			default:
+				return <Notice { ...statusProps } text={ translate( 'Waiting to install' ) } />;
```

Comment:
I think the following is fine, but if we think that we'll always pass through the same status props, we can maybe move the switch into `getStatusText` where:

```javascript
renderStatus( plugin ) {
  if ( plugin.error ) {
    return this.renderStatusError( plugin );
  }

  const statusProps = {
  //...
  };

   return (
       <Notice { ...statusProps } text={ getStatusText( plugin ) } />
   );
}
```